jolt-core/src/r1cs/constraints.rs

```diff
@@ -412,6 +445,11 @@ impl R1CSBuilder {
             smallvec![(combined_z_chunks, 1), (x, -1), (y, 1), (0, -1 * (ALL_ONES + 1))],
             smallvec![]
         ); 
+        R1CSBuilder::constr_abc(instance,
+            smallvec![(is_load_instr, 1)], 
+            smallvec![(combined_z_chunks, 1), (load_or_store_value, -1)],
+            smallvec![]
+        ); 
```

Comment:
I think we actually don't need this constraint. I changed the `is_add_instr` flag to include store/load instructions (https://github.com/a16z/Lasso/blob/jolt/common/src/rv_trace.rs#L330-L339), and for store/load instructions we set x to 0. So the constraint above for add instructions should be sufficient